ykrt/src/compile/jitc_yk/codegen/x64/rev_analyse.rs

```diff
@@ -57,6 +75,64 @@ impl<'a> RevAnalyse<'a> {
         }
     }
 
+    fn an_binop(&mut self, iidx: InstIdx, binst: BinOpInst) {
+        match binst.binop() {
+            BinOp::Add | BinOp::And | BinOp::Or | BinOp::Xor => {
```

Comment:
Have you considered doing one for `Call` where the ABI constrains the argument/return registers?

Reply:
Yes, but big_loop doesn't exercise that part, so I'll do it in the futuer.